plugins/confluence/commands/sync-meeting-notes.md

```
@@ -0,0 +1,143 @@
+2. **Create meeting notes under a specific parent:**
+   ```
+   /confluence:sync-meeting-notes DEV "Architecture Review - 2026-05-07" --parent-id 123456789
+   ```
+
+3. **Update an existing meeting page with new notes:**
+   ```
+   /confluence:sync-meeting-notes TEAM "Weekly Standup" --update 987654321
+   ```
+
+## Arguments
+| Argument | Required | Description |
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)